src/core/ui/components/primitives/input.module.css

```diff
@@ -0,0 +1,24 @@
+.label {
+  display: inline-block;
+  margin-bottom: 8px;
+}
+
+.input {
+  display: block;
+  border: 1px solid var(--slashauth-lineColor, #E5E7EB);
+  border-radius: 18px;
+  height: 52px;
+  width: 100%;
+  box-sizing: border-box;
+  padding: 16px 22px;
+}
+
+.input:focus {
+  outline: none !important;
```

Comment:
I read something somewhere that `!important` was a hack buitl for one reason -- to support US government styling. Apparently it shouldn't be used in general (but is in some cases).

Out of curiosity, why are we using it here?

Reply:
Not here! Totally forgot about it... 😆